components/automate-ui/src/app/page-components/run-summary/run-summary.component.html

```diff
@@ -27,7 +27,7 @@
   <div class="run-detail-labels flex-block">
     <div class="run-metadata">
       <div class="label">Run Duration</div>
-      <div class="value">{{ renderTime(nodeRun.startTime) }} - {{ renderTime(nodeRun.endTime) }}</div>
+      <div class="value">{{ renderTime(nodeRun.startTime) }} - {{ renderTime(nodeRun.endTime) }} UTC</div>
```

Comment:
i could also inline this using the `datetime` pipe.  Is there a performance preference?

Reply:
Can't imagine much perf differences. But yea, just using the pipe and dropping `renderTime()` would be nicer.